test/registry.jl

```diff
@@ -34,6 +34,10 @@ function provider(
             for (r, c) in info.compat
                 v in r && mergewith!(∩, comp[v], c)
             end
+            hasfield(typeof(info), :weak_compat) &&
```

Comment:
Sure, but is there a strong reason to support pre LTS? I suggest just bumping the CI matrix.